src/scales/linearScale.ts

```diff
@@ -17,6 +16,8 @@ export module Scales {
     constructor(scale: D3.Scale.LinearScale);
     constructor(scale?: any) {
       super(scale == null ? d3.scale.linear() : scale);
+      this.autoMin(-Infinity);
```

Comment:
I don't think we should "presume" things in public methods. I still think about it as "auto domain minimum... what"?